packages/backend/src/extension.ts

```diff
@@ -106,6 +81,40 @@ export async function activate(extensionContext: ExtensionContext): Promise<void
   // so the frontend can be notified when the history changes and so we can update the UI / call listHistoryInfo
   const historyNotifier = new HistoryNotifier(panel.webview, extensionContext.storagePath);
   extensionContext.subscriptions.push(historyNotifier);
+
+  extensionContext.subscriptions.push(
+    extensionApi.commands.registerCommand('bootc.image.build', async image => {
+      await openBuildPage(panel, image);
+    }),
+  );
+}
+
+export async function openBuildPage(
+  panel: extensionApi.WebviewPanel,
+  image: { name: string; tag: string },
+): Promise<void> {
+  console.log('Opening webview for ' + image.name);
+
+  // this should use webview reveal function in the future
+  const webviews = extensionApi.window.listWebviews();
+  const bootcWebView = (await webviews).find(webview => webview.viewType === 'bootc');
+
+  if (!bootcWebView) {
+    console.error('Could not find bootc webview');
+    return;
+  }
+
+  await extensionApi.navigation.navigateToWebview(bootcWebView.id);
+
+  // if we trigger immediately, the webview hasn't loaded yet and can't redirect
+  // if we trigger too slow, there's a visible flash as the homepage appears first
+  await new Promise(r => setTimeout(r, 100));
```

Comment:
Unfortunate that we're unable to figure out if the extension has loaded yet before switching to the web view. 

Has there been a solution for this within ai studio or podman desktop / they've encountered the same thing when redirecting after loading the web view?

@feloy @axel7083 @lstocchi